### PR TITLE
#166722939 Change all freckle references to noko

### PR DESCRIPTION
**Pull request by bozicschucky**
#### What does this PR do?
Changes all occurrences of Noko to freckle.

#### Description of Task to be completed?
With Freckle rebranding to Noko, this task aims at renaming all occurrences of freckle to noko.

#### How should this be manually tested?
- pull in the backend PR branch(ch-freckle-noko-167193810) for freckle to noko to get the right data from the backend.
- run the project locally  using (yarn start)and visit the different pages to see the references of freckle renamed to noko

#### Any background context you want to provide?
The backend data has to be right references else you will have breaking changes.

#### What are the relevant pivotal tracker stories?
[#166722939](https://www.pivotaltracker.com/story/show/166722939)

#### Screenshots (If applicable)

### CODE REVIEW

File: src/components/developerCards/index.jsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ class DeveloperCard extends Component {
 
   renderStatusBand = (className, status, cardId) => {
     const { retryingAutomation, handleRetryAutomation, data } = this.props;
+    // console.log('backend data ----->,',data);
```
Comment (katunold) [inline src/components/developerCards/index.jsx]: Kindly delete this commented out code

File: src/components/developerCards/index.jsx [hunk 2]
```diff
@@ -72,6 +73,7 @@ class DeveloperCard extends Component {
 
   renderCards = () => {
     const { data, openModal, changeModalTypes } = this.props;
+    console.log('api data ------>',data);
```
Comment (katunold) [inline src/components/developerCards/index.jsx]: Also, don't forget to delete all console logs when you are done with the changes you are working on

**Comment by 09arnold** [comment]
Rename `public\Freckle.png` to `public\Noko.png`

**Comment by bozicschucky** [comment]
> Rename `public\Freckle.png` to `public\Noko.png`

I worked on the change. Thanks

**Pull request merged**